### ISSUE TITLE
Deploy DASH into QA

### ISSUE THREAD

**Issue by awoods** [harvard-lts/HDC#138]

**Comment by dl-maura** [comment]
I've started this but may end up in next sprint, drs took priority

**Comment by dl-maura** [comment]
Deployed to qa, waiting on alb setup

**Comment by sbiver** [comment]
@dl-maura do we have any update on alb set up?

**Comment by dl-maura** [comment]
Its done! I will close this. We will want a new ticket for NRS on qa though, thats a separate process from this
